src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: 300ms ease;
```

Comment:
Always add a transition for styles that you want smoothly change

transition: style-name 300ms ease;